bin/plotting/pycbc_mass_area_plot

```diff
@@ -0,0 +1,109 @@
+#!/usr/bin/env python
+#
+# Integration of the area laying in the different cbc regions
+# By A. Curiel Barroso
+# August 2019
+
+"""This script computes the area corresponding to different CBC on
+the m1 & m2 plane when given a central mchirp value and uncertainty.
+"""
+
+import argparse
+from mchirp_area import calc_areas
```

Comment:
These will then need to be changed since mchirp_area is a pycbc module - i.e. `from pycbc.mchirp_area import` etc